data/crac-creation/crac-creator-csa-profiles/src/main/java/com/powsybl/open_rao/data/crac_creation/creator/csa_profile/parameters/JsonCsaCracCreationParameters.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.open_rao.data.crac_creation.creator.csa_profile.parameters;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.google.auto.service.AutoService;
+import com.powsybl.open_rao.commons.OpenRaoException;
+import com.powsybl.open_rao.data.crac_creation.creator.api.parameters.JsonCracCreationParameters;
+
+import java.io.IOException;
+
+/**
+ * @author Thomas Bouquet {@literal <thomas.bouquet at rte-france.com>}
+ */
+@AutoService(JsonCracCreationParameters.ExtensionSerializer.class)
+public class JsonCsaCracCreationParameters implements JsonCracCreationParameters.ExtensionSerializer<CsaCracCreationParameters> {
+
+    private static final String USE_CNEC_GEOGRAPHICAL_FILTER = "use-cnec-geographical-filter";
```

Comment:
this name is not very explicit. I would call it "forbid-implicit-extranational-ae-co-combinations"